config/default.schema.json

```diff
@@ -1336,28 +1336,27 @@
 			"properties": {
 				"ENABLED": {
 					"type": "boolean",
+					"default": true,
 					"description": "Flag to turn on/off the Admin API."
 				},
 				"PORT": {
 					"type": "number",
+					"default": 4030,
 					"description": "Port of the exposed Admin API server."
 				},
 				"ALLOWED_API_KEYS": {
 					"type": "string",
+					"default": "",
 					"description": "Allowed Admin API keys (for accessing the Admin API). - A comma seperated list of strings where description is optional e.g. [<description>:]<token>,[<description>:]<token>"
 				},
 				"MODIFICATION_THRESHOLD_MS": {
 					"type": "number",
+					"default": 60000,
 					"description": "threshold in milliseconds to get deletionRequest to delete"
 				}
 
 			},
-			"default": {
-				"ENABLED": true,
-				"PORT": 4030,
-				"MODIFICATION_THRESHOLD_MS": 60000,
-				"ALLOWED_API_KEYS": ""
-			}
```

Comment:
why do you remove it? I think this changes don't work for dev

Reply:
This way also not work (anymore?). Why i have no idea.